app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    command_list = command.split()
```

Comment:
You have a variable named `command_list`, which is not very descriptive. According to the checklist, you should use descriptive and correct variable names. Consider renaming this variable to something more descriptive, like `parsed_command`.